### PR TITLE
Edit an issue on Github

### PR DESCRIPTION
**Pull request by AdityaA1998**

### CODE REVIEW

File: src/main/java/seedu/progresschecker/logic/commands/EditIssueCommand.java [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+        try {
+            model.updateIssue(index, editedIssue);
+        } catch (IOException io) {
+            throw  new CommandException(io.getLocalizedMessage());
+        }
+        return new CommandResult(String.format(MESSAGE_EDIT_ISSUE_SUCCESS, index.getOneBased()));
+    }
+
+    /**
+     * Prepdfef
```
Comment (EdwardKSG) [inline src/main/java/seedu/progresschecker/logic/commands/EditIssueCommand.java]: what does it mean?

File: src/main/java/seedu/progresschecker/model/ProgressChecker.java [hunk 2]
```diff
@@ -219,23 +247,43 @@ public void closeIssueOnGithub(Index index) throws IOException, CommandException
...
+            toEdit.setMilestone(check);
+        }
+        toEdit.setTitle(editedIssue.getTitle().toString());
+        toEdit.setBody(editedIssue.getBody().toString());
+        toEdit.setAssignees(listOfUsers);
+        toEdit.setLabels(listOfLabels.toArray(new String[0]));
     }
 
+    //@@author
+
```
Comment (EdwardKSG) [inline src/main/java/seedu/progresschecker/model/ProgressChecker.java:286]: this is for?

**Review by EdwardKSG — APPROVED** [review]

File: src/main/java/seedu/progresschecker/logic/commands/EditIssueCommand.java (hunk 1 above)
Comment (kohyeeru) [inline src/main/java/seedu/progresschecker/logic/commands/EditIssueCommand.java]: What is this HAHAHA

File: src/main/java/seedu/progresschecker/model/Model.java [hunk 3]
```diff
@@ -45,6 +47,15 @@
...
     void closeIssueOnGithub(Index index) throws IOException, CommandException;
 
+    /**
+     * Replaces the fields in Issue {@code index} with {@code editedIssue}.
+     *
+     * @throws IOException if while updating the issue there is some problem in authentication
+     */
+    void updateIssue(Index index, Issue editedIssue) throws IOException;
+
+    //@@author adityaa1998
```
Comment (kohyeeru) [inline src/main/java/seedu/progresschecker/model/Model.java]: Was this meant to be blank author ```//@@author``` instead?

File: src/main/java/seedu/progresschecker/model/ProgressChecker.java (hunk 2 above)
Comment (kohyeeru) [reply src/main/java/seedu/progresschecker/model/ProgressChecker.java:286]: To set a breakpoint up till where his code contribution is.

**Review by kohyeeru — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/progresschecker/model/ProgressChecker.java (hunk 2 above)
Comment (AdityaA1998) [reply src/main/java/seedu/progresschecker/model/ProgressChecker.java:286]: yep

**Review by kohyeeru — APPROVED** [review]

**Pull request merged**